### ISSUE TITLE
Resize icons

### ISSUE THREAD

**Issue by orlandovallejos** [devias-io/untitled-ui#3]
Is there a way to resize icons from the react library?
I tried changing the width and height attributes, but this is the result:

This is the `Home01` icon, and I set the size to 20px x 20px:

<img width="146" alt="image" src="https://github.com/devias-io/untitled-ui/assets/18174459/017eb53b-8036-4881-92f2-b226c2c60af7">

**Comment by acomanescu** [comment]
@orlandovallejos you can use "font-size" css attribute to resize them.

**Comment by orlandovallejos** [comment]
@arobert93 Thanks for your answer, but I tried that before and it doesn't work.

```
import { Activity } from "@untitled-ui/icons-react";
...
...
<Activity style={{ fontSize: "30px" }} />
```

And the icon doesn't resize:
https://imgur.com/a/0g8xXpt

Have you tried it? Did it work for you? 🤔

**Comment by schulzf** [comment]
@orlandovallejos  you can do it by setting viewBox prop to "0 0 24 24" and then play with the height and width...
would be nice to make a PR with the default viewBox prop set to all the icons

**Comment by acomanescu** [comment]
Fixed and released v0.1.2
